modules/prometheus-service-monitor.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: PROCEDURE
+[id="prometheus-service-monitor-example_{context}"]
+= Monitoring Central using Prometheus service monitor
+
+[role="_abstract"]
+If you are using the Prometheus Operator, you can use the service monitor to get the metrics from {rh-rhacs-first}.
+
+[NOTE]
+====
+If you are not using the Prometheus operator, you must edit the Prometheus configuration files to receive the data from {product-title-short}.
+====
+
+.Procedure
+. Create a new `servicemonitor.yaml` file with the following content:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: prometheus-stackrox
+  namespace: stackrox
+spec:
+  endpoints:
+    - interval: 30s
+      port: monitoring
+      scheme: http
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: <stackrox-service> <1>
+----
+<1> The labels must match with the `Service` resource that you want to monitor. For example, `central`, `scanner`, or `collector`.
```

Comment:
I would not mention collector here, since we have not mentioned it in the above config flags. There are already `exposeMonitoring` flags for secured cluster services in the Helm chart, however they are missing from the operator CR. My suggestion is to come back to this doc page once the secured cluster CR has the monitoring flag as well, and then we add `sensor` and `collector` to this page.